### PR TITLE
(DOCSP-9352): Port AWS Service Overview

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
JIRA: https://jira.mongodb.org/browse/DOCSP-9352

Stage: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/DOCSP-9352/services/aws.html

### CODE REVIEW

File: source/services/aws.txt [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+
+.. tabs-realm-sdks::
+
+   tabs:
+     - id: javascript
+       content: |
+         .. code-block:: javascript
+
+            // MyAwsService is the name of the aws service you created in
+            // the realm UI, and it is configured with a rule
```
Comment (cbush) [inline source/services/aws.txt]:
```suggestion
            // the Realm UI, and it is configured with a rule
```

File: source/services/aws.txt [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+               }).catch(err => {
+                  // handle failure
+               });
+
+     - id: android
+       content: |
+         .. code-block:: java
+
+            // MyAwsService is the name of the aws service you created in
+            // the realm UI, and it is configured with a rule
```
Comment (cbush) [inline source/services/aws.txt]:
```suggestion
            // the Realm UI, and it is configured with a rule
```

File: source/services/aws.txt [hunk 3]
```diff
@@ -0,0 +1,505 @@
...
+
+You must specify rules to enable the AWS services and actions. Each rule
+applies to a single service API, plus one or all actions on that service.
+As with other service rules in Realm, a rule must evaluate
+to ``true`` to enable the action.
+
+For example, the following rule enables all actions on the Kinesis
+service:
+
+.. figure:: /images/aws-kinesis-rule.png
```
Comment (cbush) [inline source/services/aws.txt]: Are these images copied over?

Comment (nathan-contino-mongo) [reply source/services/aws.txt]: totally missed these before -- got 'em in this commit.

**Pull request merged**